### ISSUE TITLE
Add filtering functionality to blog and portfolio

### ISSUE THREAD

**Issue by ricardobalk** [ricardobalk/www#32]

**Comment by ricardobalk** [comment]
- Added this to **Milestone 3.4**, due Aug 1, 2020.
- Time needed: 120 minutes +/- 50%.
- Already made a basic filter, shown below:

```js
let filterPosts = ({ category }) => {
    Array.prototype.forEach.call(document.querySelectorAll('article[data-category]'), function(e) {
        e.style.display = (e.dataset.category == category ? '' : 'none');
    })
};
```

That will basically add `style="display: none;"` to each `article[data-category]` except the ones you'd like to see.
<small>e.g. `filterPosts({ category: "front-end-development"});` will show `article[data-category="front-end-development"]` and hide others.</small>

- This need to be turned into an interactive Vue component.

**Comment by ricardobalk** [comment]
This feature has been implemented in dfdb12ef6e2b7b66f0003f5ac743ac189c796538. It needs refactoring, but a separate issue has been opened for that (#75).
